src/bentoml/_internal/server/http_app.py

```diff
@@ -310,6 +310,8 @@ async def api_func(request: Request) -> Response:
             try:
                 input_data = await api.input.from_http_request(request)
                 ctx = None
+                # NOTE: function should always be set here, but check anyway.
```

Comment:
```suggestion
```

An assert should always mean this.

Reply:
I don't think these comments add anything, let's remove them?

Reply:
I thought I have already removed them. Let me do that now.